src/main/java/run/halo/app/handler/migrate/converter/WordPressConverter.java

```diff
@@ -17,128 +18,144 @@
 import java.time.LocalDateTime;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
+import java.util.*;
 import java.util.function.Function;
 
 /**
  * WordPress博客数据迁移转换器
+ *
  * @author guqing
  * @date 2020-01-18 16:50
  */
-public class WordPressConverter extends Converter<Rss, List<PostVO>> {
+public class WordPressConverter implements Converter<Rss, List<PostVO>> {
 
-    public WordPressConverter() {
-        super(new PostVoFunction());
+    @Override
+    public List<PostVO> converterFromDto(Rss rss) {
+        return converterFromDto(rss, this::apply);
     }
 
     /**
      * 自定义转换规则
      */
-    private static class PostVoFunction implements Function<Rss, List<PostVO>> {
-        @Override
-        public List<PostVO> apply(Rss rss) {
-            Channel channel = rss.getChannel();
-            List<Item> items = channel.getItems();
-            Assert.notNull(items, "解析出错,文章item项不能为null");
-
-            return  getBasePost(items);
+    public List<PostVO> apply(Rss rss) {
+        if(Objects.isNull(rss) || Objects.isNull(rss.getChannel())) {
+            return new ArrayList<>();
         }
 
-        private List<PostVO> getBasePost(List<Item> items) {
-            List<PostVO> posts = new ArrayList<>();
-            items.forEach(item -> {
-                PostVO postVo = new PostVO();
+        Channel channel = rss.getChannel();
 
-                // 设置文章
-                BasePost post = getBasePostFromItem(item);
-                postVo.setBasePost(post);
+        List<Item> items = channel.getItems();
+        Assert.notNull(items, "解析出错,文章item项不能为null");
```

Comment:
We do not recommend the use of `Assert#notNull` from here. Because the items is not a parameter, and this check may throw `java.lang.IllegalArgumentException`.

Reply:
挨不对啊  这里我已经改过了难道没同步过去
[WordPressConvert](https://github.com/guqing/halo/blob/migrate/src/main/java/run/halo/app/handler/migrate/converter/WordPressConverter.java)